static/app/views/dashboardsV2/widgetCard/genericWidgetQueries.tsx

```diff
@@ -312,9 +312,11 @@ class GenericWidgetQueries<SeriesResponse, TableResponse> extends Component<
         );
       })
     );
+    const rawResultsClone = cloneDeep(this.state.rawResults) ?? [];
```

Comment:
https://github.com/getsentry/sentry/pull/36100/files#diff-ee70bd5569e09724d7e442453172053a9cb8f089ce5ee972f70d9497f2134c8bL393

This logic is derived from what we used to have here